packages/bruno-app/src/utils/codegenerator/har.js

```diff
@@ -13,13 +13,22 @@ const createContentType = (mode) => {
   }
 };
 
-const createHeaders = (headers) => {
-  return headers
+const createHeaders = (headers, type) => {
+  const result = headers
     .filter((header) => header.enabled)
     .map((header) => ({
       name: header.name,
       value: header.value
     }));
+
+  // TODO in this PR, make sure that the content type application/json is added to the headers by default
```

Comment:
I don't understand why GQL Request don't have the content-type "application/json" by default, Iguess this should be setup somewhere else in the code. addin it here seems flaky